Allow to build with OpenSSL libs in non default location.

--- unbound-1.16.2/acx_nlnetlabs.m4
+++ unbound-1.16.2/acx_nlnetlabs.m4
@@ -685,7 +685,7 @@ ssldir_include="$ssldir/include"
 		if test ! -d "$ssldir/lib" -a -d "$ssldir/lib64"; then
 			ssldir_lib="$ssldir/lib64"
 		else
-			ssldir_lib="$ssldir/lib"
+			ssldir_lib="$ssldir/lib/64"
 		fi
                 break;
             fi
